Move platform dependency to Swift Package Manager

kozlowsqi opened the pull request:
This moves the platform dependency from code to swift package manager,
meaning the @available clause can be dropped in sources.

kozlowsqi committed:
Move platform dependency to Swift Package Manager

This moves the platform dependency from code to swift package manager,
meaning the @available clause can be dropped in sources.

kieranb662 commented on Package.swift:
```diff
@@ -1,30 +1,27 @@
+        .iOS(.v13),
+        .macOS(.v10_15),
+        .watchOS(.v6),
+        .tvOS(.v13)
```
Drag Gesture not available on tvOS. Other than that, looks good

kieranb662 reviewed (changes requested):
`DragGesture` not available on TVOS. Other than that great job, Thanks!

kozlowsqi commented on Package.swift:
```diff
@@ -1,30 +1,27 @@
+        .iOS(.v13),
+        .macOS(.v10_15),
+        .watchOS(.v6),
+        .tvOS(.v13)
```
```suggestion
```

kozlowsqi replied:
I suggest we remove it then from the supported platforms for now.. 
 Note that the tvOS picker looks like: 

<img width="149" alt="image" src="https://user-images.githubusercontent.com/32329188/80470193-f389df00-8941-11ea-9114-6d095c563ce7.png">

And shouldn't be too hard to support

kozlowsqi committed:
Remove tvOS from supported platforms

As tvOS isn't yet supported by this package, I removed it from the list of supported platforms.

kozlowsqi commented:
this still active?
